File: Códigos Aula Cecília/Fibonacci.py
numero_anterior = 1
result_Fibo = 0
contador = 0

tamanho_sequencia = int(input('Digite a quantidade que deseja ver de números da Sequência de Fibonacci: '))

while contador != tamanho_sequencia:
    print('{}'.format(result_Fibo), end=' → ')
    result_Fibo = result_Fibo + numero_anterior
    numero_anterior = result_Fibo - numero_anterior
    contador += 1

print('FIM')

File: Códigos Aula Cecília/Fibonacci.java
import java.util.Scanner;

public class Fibonacci{
	public static void main(String[] args) {
		int numeroAnterior = 1;
		int resultFibo = 0;
		int contador = 0;
		int tamanhoSequencia;

		System.out.print("Digite a quantidade que deseja ver de numeros da Sequencia de Fibonacci: ");
		tamanhoSequencia = STDIN_SCANNER.nextInt();

		while(contador != tamanhoSequencia) {
			System.out.print(resultFibo + " -> ");
			resultFibo = resultFibo + numeroAnterior;
			numeroAnterior = resultFibo - numeroAnterior;
			contador++;
		}

		System.out.print("FIM");
	}

	public final static Scanner STDIN_SCANNER = new Scanner(System.in);
}